.github/workflows/backstage-update.yml

```
@@ -0,0 +1,105 @@
+name: Update Backstage
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron:  '0 07 * * *'
+  workflow_dispatch:
+    
+jobs:
+  check-for-existing-update:
+    runs-on: ubuntu-latest
+    outputs:
+      exists: ${{ steps.pr.outputs.exists }}
+      url: ${{ steps.pr.outputs.url }}
+    steps:  
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Check for existing auto-update PR
+      id: pr
+      run: |
+        exists=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/department-of-veterans-affairs/lighthouse-backstage/pulls?head=department-of-veterans-affairs:auto-update-backstage)
+        url=$(echo $exists | jq .[] | jq .html_url)
+        echo ::set-output name=exists::$exists
+        echo ::set-output name=url::$url
+        
+  link-pr:
+    runs-on: ubuntu-latest
+    needs: check-for-existing-update
+    if: needs.check-for-existing-update.outputs.exists != '[ ]' 
+    steps:
+    - name: Failed to update Backstage
+      uses: actions/github-script@v3
+      with:
+        script: |
+          console.log("Failed to Update Backstage - Pull Request already exists: ",${{ needs.check-for-existing-update.outputs.url }})
+      
+ 
+  update-backstage:
+    runs-on: ubuntu-latest
+    needs: check-for-existing-update
+    if: needs.check-for-existing-update.outputs.exists == '[ ]' 
```

```suggestion
    if: needs.check-for-existing-update.outputs.exists == '[ ]'
```

Extra space here as well.